tests/test_internals.py

```diff
@@ -10,6 +9,11 @@
                        _cache_invalidate, _cache_miss, _close, _close_waited,
                        _done_callback, _open, _wait_closed, create_future)
 
+try:
+    from test.test_asyncio import utils as test_utils
+except ImportError:
+    from asyncio import test_utils
```

Comment:
The import order must be different. First, import what might be missing, then, fallback to 100% present backup plan.